ckanext/qgov/common/plugin.py

```diff
@@ -130,6 +130,13 @@ def related_update(context, data_dict):
     return {'success': False,
             'msg': _('You must be logged in and have permission to create datasets to update a related item')}
 
+def auth_user_list(context, data_dict):
+    """Check whether access to the user list is authorised.
+    Restricted to sysadmins as per QOL-5710.
+    """
+    user = context.get('user')
+    return {'success': authz.is_sysadmin(user)}
```

Comment:
can't we allow editors to see as well instead of just sysadmin which is very small?

Reply:
See https://servicesmadesimpler.govnet.qld.gov.au/jira/browse/QOL-5710